src/ui_dialogs.ts

```diff
@@ -42,15 +58,51 @@ export function makePopup(attachement:Selection<any>, title:string, body:string)
       left: pos.left + 'px',
       top: pos.top + 'px'
     }).html(dialogForm(title, body));
+
   function movePopup() {
     //.style("left", (this.parentElement.offsetLeft + (<any>event).dx) + 'px')
     //.style("top", (this.parentElement.offsetTop + event.dy) + 'px');
     //const mouse = d3.mouse(this.parentElement);
     $popup.style({
       left: (this.parentElement.offsetLeft + (<any>d3event).dx) + 'px',
-      top:  (this.parentElement.offsetTop + (<any>d3event).dy) + 'px'
+      top: (this.parentElement.offsetTop + (<any>d3event).dy) + 'px'
     });
   }
+
+  $popup.select('span.lu-popup-title').call(behavior.drag().on('drag', movePopup));
+  $popup.on('keydown', () => {
+    if ((<KeyboardEvent>d3event).which === 27) {
+      $popup.remove();
+    }
+  });
+  var auto = <HTMLInputElement>$popup.select('input[autofocus]').node();
+  if (auto) {
+    auto.focus();
+  }
+  return $popup;
+
+}
+
+export function makesortPopup(attachement: Selection<any>, title: string, body: string) {
```

Comment:
Yes but I do not need the undo tick mark and cross mark button since I am using radio button for sort. Is there another way to reuse the default makePopup without undo, tickmark and cross mark button ?